migration/instructions-for-infra-team.md

```diff
@@ -35,7 +35,7 @@ GH-LUCENE-10677.json
 # set the GitHub PAT token to an env variable
 migration $ cp .env.example .env
 migration $ vi .env
-export GITHUB_PAT=<set the personal access token to be used for importing here>
+export GITHUB_PAT=<set the `asfgit` personal access token to be used for importing here>
```

Comment:
Note: `asfbot` is my proposal/request to infra. I think we are still not fully sure what account will be used, but fine with this for now. I'll make sure of the correct account information.